### PR TITLE
Native install postgres.app updates

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
This PR updates the instructions for installing Postgres 11 and PostGIS 2.5 via the Postgres.app. Installing PostGIS via Pex may be unfeasible and the patch instructions are outdated.

### CODE REVIEW

File: docs/setup/native.md [hunk 1]
```diff
@@ -90,6 +90,12 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `brew install redis`
 1. Install Postgresql & PostGIS
    1. It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) which installs the correct combination of Postgresql and PostGIS versions.
+   	  - Navigate to: https://postgresapp.com/downloads.html
+	  - Download the Postgres.app with PostgreSQL 10, 11 and 12
+	  - OSX: After download is complete, move the Postgres.app into your applications folder 
+	  - Once moved into the applications folder, create a new server with PostgreSQL version 11 selected
```
Comment (johnpaulashenfelter) [inline docs/setup/native.md:96]: I'd just point them to the install docs from posgresapp - i think a developer who's going to work on the platform should be able to get the app installed.

The note on which versions to chose are important for sure.

File: docs/setup/native.md [hunk 2]
```diff
@@ -90,6 +90,12 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `brew install redis`
 1. Install Postgresql & PostGIS
    1. It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) which installs the correct combination of Postgresql and PostGIS versions.
+   	  - Navigate to: https://postgresapp.com/downloads.html
```
Comment (johnpaulashenfelter) [inline docs/setup/native.md:93]: This line seems to duplicate the link above

File: docs/setup/native.md (hunk 1 above)
Comment (LindseySaari) [reply docs/setup/native.md]: Updated ✔️

File: docs/setup/native.md (hunk 2 above)
Comment (LindseySaari) [reply docs/setup/native.md]: Updated ✔️

File: docs/setup/native.md [hunk 3]
```diff
@@ -90,6 +90,8 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `brew install redis`
 1. Install Postgresql & PostGIS
    1. It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) which installs the correct combination of Postgresql and PostGIS versions.
+	  - Download the Postgres.app with PostgreSQL 10, 11 and 12
+	  - Install Instructions here: https://postgresapp.com/
```
Comment (LindseySaari) [inline docs/setup/native.md:94]: @johnpaulashenfelter Updated ✔️

**Pull request merged**